setup.py

```diff
@@ -25,9 +25,16 @@
 
 # Testing dependency groups.
 testing_group = set(extras.keys()) - {"accept-rom-license", "atari"}
-extras["testing"] = list(
-    set(itertools.chain.from_iterable(map(lambda group: extras[group], testing_group)))
-) + ["pytest==7.0.1"]
+extras["testing"] = (
+    list(
+        set(
+            itertools.chain.from_iterable(
+                map(lambda group: extras[group], testing_group)
+            )
+        )
+    )
+    + ["pytest==7.0.1", "gym>=0.26"]
```

Comment:
@pseudo-rnd-thoughts 
Yes, I thought I added the dependency in this line, but apparently no looking at the tests
Do you know why?

Reply:
Python 3.6 has a custom build requirement, see py.Dockerfile line 29

Reply:
Oh, I see, thanks